tests/unit/lib/build_module/test_bundler.py

```diff
@@ -240,3 +241,22 @@ def test_update_function_handler(self):
         self.assertEqual(updated_handler_a, "app.handler")
         self.assertEqual(updated_handler_b, "app.handler")
         self.assertEqual(updated_handler_c, "functions/source/update/app.handler")
+
+    @parameterized.expand(
+        [("/opt/my/path/handler.handler", "/opt/my/path/handler.js"), ("handler.handler", "handler.js")]
+    )
+    def test_get_handler_path_unix(self, input_path, expected_path):
+        result_path = EsbuildBundlerManager(Mock())._get_path_and_filename_from_handler(input_path)
+
+        self.assertEqual(result_path, expected_path)
+
+    @parameterized.expand(
+        [
+            ("\\opt\\my\\path\\handler.handler", "/opt/my/path/handler.js"),
+        ]
+    )
+    @skipIf(not IS_WINDOWS, "Skipping POSIX converting logic since WindowsPath is not available on unix systems")
```

Comment:
unix systems don't have the `WindowsPath` object, making things a bit difficult to test. Instead, I just skipped this unit test if its not running on Windows.

Reply:
I got confused .. are we going to skip running this test case in windows or we want to run it only on windows. this condition `not IS_WINDOWS` is confusing me.

Reply:
We'd want to only run if it is Windows. The `IS_WINDOWS` constant when used with a `not` in the `skipIf` does make it a little weird to read out.